src/components/Shortcuts/ClipboardShortcuts/ClipboardShortcuts.test.tsx

```diff
@@ -118,21 +124,24 @@ describe('<CopyAndPasteShortcuts>', () => {
     expect(clipboardData.setData).not.toBeCalled();
   });
 
-  it('should cut content', () => {
+  it('should cut contents', () => {
     const activeSlide = activeWhiteboardInstance.getSlide('slide-0');
 
     render(<ClipboardShortcuts />, { wrapper: Wrapper });
 
     const clipboardData = fireClipboardEvent('cut');
 
-    expect(clipboardData.setData).toBeCalledWith('text/plain', 'Hello World');
+    expect(clipboardData.setData).toBeCalledWith(
+      'text/plain',
+      'Hello World 1 Hello World 2',
+    );
     expect(clipboardData.setData).toBeCalledWith(
       'text/html',
       expect.stringContaining('<span data-meta='),
     );
 
-    expect(activeSlide.getActiveElementId()).toBe(undefined);
-    expect(activeSlide.getElement('element-1')).toBeUndefined();
+    expect(activeSlide.getActiveElementIds()).toHaveLength(0);
```

Comment:
Across tests it is common to check it in a bit more explicit way:
```suggestion
    expect(activeSlide.getActiveElementIds()).toEqual([]);
```